IBAnimatable-Material/AnimatableMaterialLabel.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  AnimatableMaterialLabel.swift
+//  IBAnimatable-Material
+//
+//  Created by George Kye on 2016-07-18.
+//  Copyright © 2016 IBAnimatable. All rights reserved.
+//
+
+import Foundation
+import IBAnimatable
+import Material
+
+@IBDesignable
+public class AnimatableMaterialLabel: MaterialLabel, CornerDesignable, FillDesignable, Animatable, RotationDesignable, BorderDesignable {
+  
+  // MARK: - CornerDesignable
+  @IBInspectable public var cornerRadius: CGFloat = CGFloat.NaN {
+    didSet {
+      configCornerRadius()
+    }
+  }
+  
+  // MARK: - FillDesignable
+  @IBInspectable public var fillColor: UIColor? {
+    didSet {
+      configFillColor()
+    }
+  }
+  
+  @IBInspectable public var predefinedColor: String? {
+    didSet {
+      configFillColor()
+    }
+  }
+  
+  @IBInspectable public var opacity: CGFloat = CGFloat.NaN {
+    didSet {
+      configOpacity()
+    }
+  }
+  
+  // MARK: - BorderDesignable
+  @IBInspectable public var borderColor: UIColor? {
+    didSet {
+      configBorder()
+    }
+  }
+  
+  @IBInspectable public var borderWidth: CGFloat = CGFloat.NaN {
+    didSet {
+      configBorder()
+    }
+  }
+  
+  @IBInspectable public var borderSide: String? {
+    didSet {
+      configBorder()
+    }
+  }
+  
+  // MARK: - Animatable
+  @IBInspectable public var animationType: String?
+  @IBInspectable public var autoRun: Bool = true
+  @IBInspectable public var duration: Double = Double.NaN
+  @IBInspectable public var delay: Double = Double.NaN
+  @IBInspectable public var damping: CGFloat = CGFloat.NaN
+  @IBInspectable public var velocity: CGFloat = CGFloat.NaN
+  @IBInspectable public var force: CGFloat = CGFloat.NaN
+  @IBInspectable public var repeatCount: Float = Float.NaN
+  @IBInspectable public var x: CGFloat = CGFloat.NaN
```

Comment:
`MaterialLabel` doesn't have `x` and `y`, but we are going to remove them from IBAnimatable.